src/components/victory-scatter/victory-scatter.js

```diff
@@ -438,6 +452,8 @@ export default class VictoryScatter extends React.Component {
   }
 
   render() {
+    this.props = Object.assign({}, this.props, Size.getWidthHeight(this.props,
```

Comment:
You should never modify props like this. Create a new object instead, or just pass in the `width` and `height` where you need them like `<VictoryScatter {...this.props} {width, height} />`